src/main/java/core/basesyntax/service/WriteDataToFile.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface WriteDataToFile {
+    void writeReportToFile(String toFileName, String stringReport);
+}
```

Comment:
The same WriterService can write to file anything 
```suggestion
public interface WriterService {
    void writeToFile(String toFileName, String data);
}
```